.github/workflows/release.yml

```diff
@@ -43,6 +43,18 @@ jobs:
           git commit -m "Release extension version ${{ inputs.version }}"
 
           git push --set-upstream origin release/${{ inputs.version }}
+      
+      - name: Update language server
+        if: ${{ inputs.update-language-server }}
+        run: |
+          npm install @actions/languageserver@latest @actions/workflow-parser@latest --workspaces=false
```

Comment:
Why are we also installing the `workflow-parser` here?

Reply:
Because its the [manual step](https://github.com/github/workflow-development/blob/main/docs/vscode/release-new-version.md#local-changes-to-bump-version-and-edit-changelog) given in the readme and i copied it exactly :)